modules/ztp-manually-install-a-single-managed-cluster.adoc

```diff
@@ -51,22 +51,20 @@ provisioner_cluster_registry }}/ocp4:{{ mirror_version_spoke_release }}
 
 * You mirrored the ISO and `rootfs` used to generate the spoke cluster ISO to an HTTP server and configured the settings to pull images from there.
 +
-The images must match the version of the `ClusterImageSet`. To deploy a 4.9.0 version, the `rootfs` and
-ISO must be set at 4.9.0.
-
+The images must match the version of the `ClusterImageSet`. For example, to deploy a {product-version}.0 version, the `rootfs` and ISO must be set to `{product-version}.0`.
 
 .Procedure
 
 . Create a `ClusterImageSet` for each specific cluster version that needs to be deployed. A `ClusterImageSet` has the following format:
 +
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
 apiVersion: hive.openshift.io/v1
 kind: ClusterImageSet
 metadata:
-  name: openshift-4.9.0-rc.0 <1>
+  name: openshift-{product-version}.0-rc.0 <1>
```

Comment:
please remove "-rc.0" from the name, so it matches with the releaseImage defined below.

Reply:
fixed